sympy/polys/matrices/domainmatrix.py

```diff
@@ -21,6 +21,47 @@
 
 
 class DomainMatrix:
+    r"""
+        The idea behind DomainMatrix is to associate :py:class:`~.Domain` with matrix.
+
+        Explanation
+        ===========
```

Comment:
Everything in the docstring should have the same indentation as the opening quotes. See
https://docs.sympy.org/latest/documentation-style-guide.html